docs/appendices/release-notes/5.5.1.rst

```diff
@@ -49,4 +49,5 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Added restriction to not allow to drop a column which participates in a
+  :ref:`table level constraint with other columns <check_constraint_multiple_cols>`.
```

Comment:
So we should phrase what change is visible for the user instead?
```suggestion
- Return meaningful error when trying to drop a column which participates
  in a
  :ref:`table level constraint with other columns <check_constraint_multiple_cols>`.
```

Reply:
ok, yes, makes sense, thx!

Reply:
fixed